### DISCUSSION TITLE
Projects using taskiq

### DISCUSSION THREAD

**Discussion by s3rius** [taskiq-python/taskiq#69] [General]
If you're using taskiq and want to say warm thanks, please write a message here and describe your experience if it's possible. With every message in this discussion we gain motivation to make project even better.

**Comment by lembregtse** [comment]
We are integrating taskiq (aio_pika, redis, pipelines and dependencies) for our multiple worker environments. Using about 10 different brokers and about 100 tasks. 

We are migrating from celery because we love the lean and mean approach and extendibility of taskiq.

**Reply by s3rius** [reply]
We really appreciate it. Thanks for your support.

**Reply by devraj** [reply]
Likewise, I am almost done migrating all projects at work.

Thank you for your work and if you start accepting sponsorships on Github, we would be happy to support your work

**Reply by s3rius** [reply]
@devraj , thank you very much. Acutally, we already have an [opencollective](https://opencollective.com/taskiq-python) account, but we wanted to add more info about the project before adding sponsorship button. But it's better to wait until we add sponsorship button, so you will get your your sponsor badge.

**Reply by s3rius** [reply]
For anyone who was interested in sponsorship. Finally, Github has approved the sponsorship program for our project. Now you can find sponsorship button in our main repo or on organization page.

Thanks for your interest to this project and your support. 

All the best, your taskiq team.

**Reply by devraj** [reply]
Nice. Just added support from our business. Thanks for all the great work.
